PowerEditor/src/ScintillaComponent/UserDefineDialog.cpp

```diff
@@ -925,9 +925,14 @@ void UserDefineDialog::reloadLangCombo()
 void UserDefineDialog::changeStyle()
 {
     _status = !_status;
-    ::SetDlgItemText(_hSelf, IDC_DOCK_BUTTON, (_status == DOCK)?TEXT("Undock"):TEXT("Dock"));
-
-	auto style = ::GetWindowLongPtr(_hSelf, GWL_STYLE);
+    NativeLangSpeaker *pNativeSpeaker = (NppParameters::getInstance()).getNativeLangSpeaker();
+    TiXmlNodeA *targetNode = nullptr;
+    pNativeSpeaker->getNativeLangA() && (targetNode = (pNativeSpeaker->getNativeLangA())->FirstChildElement("Dialog")) && (targetNode = targetNode->FirstChildElement("UserDefine"));

```

Comment:
C:\projects\notepad-plus-plus\PowerEditor\src\ScintillaComponent\UserDefineDialog.cpp(930): error C2220: the following warning is treated as an error [C:\projects\notepad-plus-plus\PowerEditor\visual.net\notepadPlus.vcxproj]
     5>C:\projects\notepad-plus-plus\PowerEditor\src\ScintillaComponent\UserDefineDialog.cpp(930): warning C4706: assignment within conditional expression [C:\projects\notepad-plus-plus\PowerEditor\visual.net\notepadPlus.vcxproj]

Reply:
@chcg 
Is there any equivalent in GCC for this warning ([manual](https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#Warning-Options))? For `Debug` we turn on a lot of things but I don't see any such message: https://ci.appveyor.com/project/donho/notepad-plus-plus/builds/46232207/job/8aqqrg6cde100t7m.